fotoapparat/src/main/java/io/fotoapparat/view/CameraView.java

```diff
@@ -52,8 +57,21 @@ public CameraView(@NonNull Context context,
 
     private void init() {
         rendererView = new TextureRendererView(getContext());
-
         addView(rendererView);
+
+        focusMarkerLayout = new FocusMarkerLayout(getContext());
+        focusMarkerLayout.setOnTouchListener(new OnTouchListener() {
```

Comment:
The `onTouch` logic should go to `FocusMarkerLayout` itself. Just override `onTouchEvent` method there.